### ISSUE TITLE
Allow users to easily select all data layers under a group/sub-group 

### ISSUE THREAD

**Issue by AnaBelgun** [TerriaJS/terriajs#6684]
Allow users to enable all layers under a group or sub-group in the catalogue to the workbench Post adding, it should allow users to disable layers. 
Triggered by the need to create a complete 'image' of an area (made up from different reality meshes, buildings, or other data, including 2D). 

Options for implementation: 
- [ ] enable a button under group / subgroup to allow users to "select all" 
- [ ] use a 'composite' layer made of different other layers (not necessarily in the same group)

**Comment by AnaBelgun** [comment]
Update 1 Feb 2023: 
- Use an icon showing a PLUS sign with a list for "Add all" or "Enable all" (for adding all layers under a sub-group/group)
- Use an icon showing a MINUS sign for the same list for "Remove all" or "Disable all) (for removing all layers under the same sub-group/group) 
- if any particular one has been removed, the button reverts to "add all" 
- to be applied manually for groups or sub-groups that don't impact the user experience (performance etc) using the 'trait' method enabled / disabled for those group
- to check how we handle broken datasets - error message for the ones not working 

@philipgrimmett to refine the design.

**Comment by philipgrimmett** [comment]
**Proposed workflow**

https://www.figma.com/proto/jtdgEiY0gxtZlGtxSJTzcZ/Terria---2023?page-id=10987%3A46025&node-id=10987%3A46026&viewport=499%2C490%2C0.22&scaling=min-zoom

**Concept images**

<img width="1461" alt="Screenshot 2023-02-07 at 3 49 50 pm" src="https://user-images.githubusercontent.com/5292858/217151107-47af233d-7848-4450-a280-bd62faa5e287.png">

<img width="1456" alt="Screenshot 2023-02-07 at 3 50 29 pm" src="https://user-images.githubusercontent.com/5292858/217151189-d711b4a3-9edb-4146-84ef-1280ff6b869c.png">

**Notes**

- If a catalogue group contains both regional and point data we need to consider the order in which we add items to the map, so that point data is not obscured
- Related to the above point: when I add a group of datasets to the map, I expect them to appear in the workbench in the same order that they appeared in the catalogue.

**Comment by staffordsmith83** [comment]
Dear @philipgrimmett this point is not yet implemented, waiting on an icon from you if you would like:
"Use an icon showing a MINUS sign for the same list for "Remove all" or "Disable all) (for removing all layers under the same sub-group/group)"

Otherwise this is merged into main

**Comment by AnaBelgun** [comment]
Update 15 March 
- to include the "remove all" new icon from Phil
